crates/chisel/bin/main.rs

```diff
@@ -183,11 +188,11 @@ async fn main_args(args: Chisel) -> eyre::Result<()> {
                 DispatchResult::CommandFailed(e) => sh_err!("{e}")?,
                 _ => panic!("Unexpected result! Please report this bug."),
             }
-            return Ok(())
+            return Ok(());
```

Comment:
Please undo `;`'s - usually inserted by the formatter being ran w/ `stable` instead of `nightly`

```
cargo +nightly fmt -- --check
```